src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -109,8 +110,8 @@ const BioAndDMSetting = ({
           value={isAnonymity}
           onValueChange={toggleSwitchAnon}
           labelLeft="Allow anonymous messages?"
-          circleActiveColor={colors.blue1}
-          activeTextColor={colors.blue1}
+          circleActiveColor={COLORS.signed_primary}
+          activeTextColor={COLORS.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The active color and text color for the toggle switch have been updated to use the new `COLORS.signed_primary` value.
```diff
- circleActiveColor={colors.blue1}
- activeTextColor={colors.blue1}
+ circleActiveColor={COLORS.signed_primary}
+ activeTextColor={COLORS.signed_primary}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->